src/ngraph/builder/reshape.hpp

```diff
@@ -0,0 +1,63 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "ngraph/node.hpp"
+#include "ngraph/shape.hpp"
+
+namespace ngraph
+{
+    namespace builder
+    {
+        /// \brief      Change shape of input tensor.
+        ///
+        /// \param[in]  node   The node which shape will be used as input to Reshape.
```

Comment:
The node _containing_ the shape to be used as input to `Reshape`?

Reply:
Neither :) You're right, this phrasing is a bit awkward: something like `The node producing the tensor to be reshaped`?